cartridges/int_alma/cartridge/scripts/helpers/almaPaymentHelper.js

```diff
@@ -184,7 +184,7 @@ function createOrderFromBasket() {
 
     Transaction.wrap(function () {
         currentBasket.removeAllPaymentInstruments();
-        var paymentProcessor = PaymentMgr.getPaymentMethod('ALMA').paymentProcessor;
+        var paymentProcessor = PaymentMgr.getPaymentMethod('ALMA_CREDIT').paymentProcessor;
```

Comment:
Is it a bug fix ?

Don't we have constant for this string ?

Reply:
could you remove this from the PR.
we will make a specific PR for this fix

Reply:
It's done.